scripts/globals/combat/physical_utilities.lua

```diff
@@ -71,6 +95,315 @@ local elementalBelt = -- Ordered by element.
     xi.items.SHADOW_BELT
 }
 
+-- List of zones in which level correction still happens.
+local pDifZoneList =
```

Comment:
Level correction also applies for magic evasion/magic accuracy

this could be moved to a different file later